file-formats/sito/type/zif_aff_sito_v1.intf.abap

```diff
@@ -0,0 +1,551 @@
+INTERFACE zif_aff_sito_v1
+  PUBLIC.
+
+  "! <p class="shorttext">SIT2_DO_SCOPE</p>
+  "! SIT2_DO_SCOPE
+  "! $values {@link zif_aff_sito_v1.data:co_sit2_do_scope}
+  TYPES ty_sit2_do_scope TYPE n LENGTH 2.
+
+  CONSTANTS:
+    "! <p class="shorttext">SIT2_DO_SCOPE</p>
+    "! SIT2_DO_SCOPE
+    BEGIN OF co_sit2_do_scope,
+      "! <p class="shorttext">SAP</p>
+      "! SAP
+      sap      TYPE ty_sit2_do_scope VALUE '01',
+      "! <p class="shorttext">Customer</p>
+      "! Customer
+      customer TYPE ty_sit2_do_scope VALUE '02',
+    END OF co_sit2_do_scope.
+
+  "! <p class="shorttext">SIT2_DO_OBJ_STRUC_TYPE</p>
+  "! SIT2_DO_OBJ_STRUC_TYPE
+  "! $values {@link zif_aff_sito_v1.data:co_sit2_do_obj_struc_type}
+  TYPES ty_sit2_do_obj_struc_type TYPE n LENGTH 2.
+
+  CONSTANTS:
+    "! <p class="shorttext">SIT2_DO_OBJ_STRUC_TYPE</p>
+    "! SIT2_DO_OBJ_STRUC_TYPE
+    BEGIN OF co_sit2_do_obj_struc_type,
+      "! <p class="shorttext">CDS</p>
+      "! CDS
+      cds       TYPE ty_sit2_do_obj_struc_type VALUE '01',
+      "! <p class="shorttext">In Memory</p>
+      "! In Memory
```

Comment:
Usually, the tile stays in title case. The descriptions below are in standard sentence case.

```suggestion
      "! <p class="shorttext">In Memory</p>
      "! In memory
```

This seems to be the case for almost all descriptions

Reply:
Adapted for all descriptions